test/cri-containerd/container_downlevel_test.go

```diff
@@ -0,0 +1,41 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"testing"
+
+	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func Test_CreateContainer_DownLevel_WCOW_Hypervisor(t *testing.T) {
```

Comment:
Can you limit this test to only run on 19H1 like we do the others?

Reply:
How are you limiting tests that should only run on 19H1? Any example test case will help.

Reply:
Sorry I didnt see this. 

Put this very first in the test:
`testutilities.RequiresBuild(t, osversion.V19H1)`

Reply:
And btw this is >= it will run on 20H1 etc.

Reply:
@jterry75 Done.